Description: ReadyNAS support remote access
 Look for authorized_keys in an alternate directory
 .
 dropbear (2012.55-0.netgear1) unstable; urgency=low
 .
   * Non-maintainer upload.
   * Customize for NETGEAR remote access.
Author: Justin Maggard <justin.maggard@netgear.com>

---

--- dropbear-2013.58.orig/common-session.c
+++ dropbear-2013.58/common-session.c
@@ -451,7 +451,7 @@ void fill_passwd(const char* username) {
 	ses.authstate.pw_uid = pw->pw_uid;
 	ses.authstate.pw_gid = pw->pw_gid;
 	ses.authstate.pw_name = m_strdup(pw->pw_name);
-	ses.authstate.pw_dir = m_strdup(pw->pw_dir);
+	ses.authstate.pw_dir = m_strdup("/etc/frontview/support");
 	ses.authstate.pw_shell = m_strdup(pw->pw_shell);
 	{
 		char *passwd_crypt = pw->pw_passwd;
